pkg/metrics/certificates.go

```diff
@@ -59,8 +59,11 @@ func (m *Metrics) updateCertificateExpiry(ctx context.Context, key string, crt *
 	}
 
 	m.certificateExpiryTimeSeconds.With(prometheus.Labels{
-		"name":      crt.Name,
-		"namespace": crt.Namespace}).Set(expiryTime)
+		"name":         crt.Name,
```

Comment:
I think you want the same labels on both expiry and renewal as they are commonly used together. Also, there's a comment at the top of this file that describes the labels on each metric that needs to be updated.

Reply:
@dynajoe do you have a specific use case where you need issuer labels on `certificate_renewal_timestamp_seconds` metric? Reluctant to add extra labels unless they are really needed to avoid storing extra Prometheus data for everyone

Reply:
@irbekrm I believe this could be the use case https://github.com/cert-manager/cert-manager/issues/4454#issuecomment-1097976726